.github/policies/resourceManagement.yml

```diff
@@ -77,18 +77,10 @@ configuration:
       - labelAdded:
           label: breaking-change
       then:
-      - addLabel:
-          label: needs-breaking-change-doc-created
       - addReply:
           reply: >-
-            Added `needs-breaking-change-doc-created` label because this issue has the `breaking-change` label. 
-
-
-            1. [ ] Create and link to this issue a matching issue in the dotnet/docs repo using the [breaking change documentation template](https://aka.ms/dotnet/docs/new-breaking-change-issue), then remove this `needs-breaking-change-doc-created` label.
```

Comment:
This removes the instructions on creating breaking change docs and links to the main doc. That main doc though says that adding the label will trigger a message with instructions to create documentation. I think you need to keep the link to dotnet/docs in place.

Reply:
This action is not relevant for a proposed breaking change that is never implemented, which is why I'm only leaving it in the PR rule. A PR is also open on the main doc to match this change: https://github.com/dotnet/sdk/pull/47797